.github/workflows/tests.yml

```diff
@@ -40,13 +43,14 @@ jobs:
           core: true
           cask: false
           test-bot: true
+          stable: ${{ matrix.branch == 'stable' }}
```

Comment:
I may need to adjust https://github.com/Homebrew/actions/blob/master/setup-homebrew/main.sh#L146 since we do have tags that aren't linked to `master` branch.

Reply:
Yeah 4.2.14 is a special out-of-band release so `master` being on 4.2.13 is intentional. Such scenarios are rare.

So that output is correct.

Reply:
I was wondering if might miss latest tag in current action since we do `git fetch --force origin` rather than `git fetch --force --tags origin` (like in `brew` - https://github.com/Homebrew/brew/blob/master/Library/Homebrew/cmd/update.sh#L63)

Reply:
Oh right, that's probably because it's prefetched in a Docker container:

Reply:
If there isn't much overhead, I can just add `--tags` to existing command or just run `git_retry -C "$HOMEBREW_REPOSITORY" fetch --force --tags origin` right before `git tag`.

Another option I considered was `brew update-reset "$HOMEBREW_REPOSITORY"` but didn't get chance to check on full impact of command.

Reply:
Adding `--tags` to the existing command makes sense